JSDemos/Demos/Form/Validation/Angular/app/app.component.html

```diff
@@ -9,7 +9,7 @@
     validationGroup="customerData"
   >
     <dxi-item itemType="group" caption="Credentials">
-      <dxi-item dataField="Email">
+      <dxi-item dataField="Email" [editorOptions]="emailOptions">
```

Comment:
Don't u want to add `editor` everywhere to make this more clear? Now "email" options seems to be confusing for me
```suggestion
      <dxi-item dataField="Email" [editorOptions]="emailEditorOptions">
```